.github/workflows/cifuzz.yaml

```diff
@@ -13,12 +13,16 @@ jobs:
     steps:
     - name: Checkout
       uses: actions/checkout@v3
+    - name: Setup Go
```

Comment:
Since we're running inside a container do we actually need this?

Reply:
Without Go we would not have `go env GOMODCACHE`. I leaned on this for consistency as well, given that other `fluxcd` repos require this due to [harder dependencies](https://github.com/fluxcd/source-controller/blob/6d479e559babb6399ff0b6f8a4fc336eb0d95421/Makefile#L183).